lib/chatbase.test.js

```diff
@@ -0,0 +1,41 @@
+const { Pool } = require('pg');
+const mocha = require('mocha')
+const chai = require('chai')
+const should = chai.should()
+const Chatbase = require('./index')
+
+describe('Chatbase Postgres', () => {
+  const mockMessage = {
```

Comment:
Where did you get this format?

Reply:
The format is put(key, value) where the value is the JSON data itself. Therefore "message" in your case will be the string data, and this client will be responsible for adding the timestamp.

Reply:
Maybe for now add timestamp as an optional parameter with default Date.now()  -- 

```
put(key, value, timestamp = Date.now())
```

I'm thinking we should actually store timestamp outself on reception in Kafka...

Reply:
I take it back. Let's change the format. Make timestamp required: 

```
put(key, value, timestamp)
```